lib/screens/landing_screen.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class LandingPage extends StatelessWidget {
+  const LandingPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Could be moved out and put into login signup widgets, since this looks like the big green signup button